packages/server/dataloader/jiraServerLoaders.ts

```diff
@@ -0,0 +1,65 @@
+import DataLoader from 'dataloader'
+import RootDataLoader from './RootDataLoader'
+import sendToSentry from '../utils/sendToSentry'
+import TaskIntegrationManagerFactory from '../integrations/TaskIntegrationManagerFactory'
+
+export interface JiraServerIssueKey {
+  teamId: string
+  userId: string
+  issueId: string
+  providerId: string
+}
+
+interface JiraServerIssue {
+  id: string
+  self: string
+  key: string
+  descriptionHTML: string
+  summary: string
+  description: string | null
+  project: {
+    key: string
+  }
+  type: 'JiraServer'
```

Comment:
-1 we should keep the spelling of `IntegrationProviderServiceEnum` 'jiraServer'

Reply:
@Dschoordsch fixed